INSTALL.txt

```diff
@@ -138,18 +160,18 @@ together toolchains for that option. The toolchains are available at
 https://bitbucket.org/carlkl/mingw-w64-for-python/downloads. The site.cfg
 should be configured like so:
```

Comment:
There's more extensive documentation on how to build at http://scipy.org/scipylib/building/index.html already, so this is anyway a bit double.